web/src/engine/websites/legacy/MangaNexus.ts

```diff
@@ -1,73 +1,97 @@
-// Auto-Generated export from HakuNeko Legacy
```

Comment:
There are various edits in this file, but it is still in legacy folder. Is it by intention (or maybe this PR is still work in progress)?

Reply:
File not yet reviewed...

Reply:
I said in PR that code "works" but website is broken and we cant get pictures (403 errors in Firefox). Therefore, i left it in legacy folder, and i didnt make e2e tests yet. I included it in the commit anyway, in order not to lose it.